test/functorch/test_aotdispatch.py

```diff
@@ -320,10 +320,11 @@ def verify_aot_autograd(
                 if isinstance(ref_o, torch.Tensor):
                     self.assertEqual(ref_o.requires_grad, test_o.requires_grad)
                     self.assertEqual(ref_o.is_leaf, test_o.is_leaf)
-                    if ref_o.requires_grad:
-                        # _is_view() should probably unconditionally be the same,
-                        # but in practice I don't think this matters for tensors that don't require grad
-                        self.assertEqual(ref_o._is_view(), test_o._is_view())
+                    # See Note [Intermediate Bases Optimization]
+                    # In cases where we know that an output's view-ness is safe to hide from autograd
+                    # (the output is a view of an intermediate that doesn't escape the graph),
+                    # we hide the view-ness from autograd.
+                    # self.assertEqual(ref_o._is_view(), test_o._is_view())
```

Comment:
We should still be able to test this no?
In particular, check that if ref_o is a view of an input or another output, we preserve that ?

Reply:
that test is definitely better, thanks - I'll add it.